lib/smart_answer_test_helper.rb

```diff
@@ -11,10 +11,24 @@ def self.responses_and_expected_results
     Dir[data_path.join('*-responses-and-expected-results.yml')]
   end
 
+  def self.configurations
+    yaml = File.read(data_path.join('configurations.yml'))
+    YAML.load(yaml)
+  end
+
+  def self.default_configuration
+    configurations.fetch('default')
+  end
+
```

Comment:
As you suggested, I've added more context to the commit notes in the commits in #2518.

* 927b3e55f06e1d27ffe642c7ed8935120faa0bf9
* 471152d22beb34afb760ed0a65f7293ef174e728